### PR TITLE
Add --linux-flavours option to set the live-config kernel

### PR DESCRIPTION
**Pull request by bluca**
Add support for modifying auto/config on the fly to set a custom
linux kernel flavours for the image.

Need this to be able to switch flavours without changing build-iso code.

@JammyStuff @DStape @jblunck

### CODE REVIEW

**Comment by jblunck** [comment]
Why is that necessary? Usually you would have a different config to generate different images. Can you please explain the background of this PR?

**Comment by bluca** [comment]
Is there any other way to generate a different per-image setting? As far as I can see there's only the livebuild_pre_run script but it's used by all images

The idea is to be able, through the repack, to have different images with different kernels without having to change the code in build-iso, same as we do for package lists, but given the flavour is in the auto/config file it's not enough to have a new pkg list

**Comment by bluca** [comment]
Tested on build-staging: https://build-staging.eng.vyatta.net/package/show/rsyslog/livecd-bvnosbase-master

Also for the love of all that is good in the world please check my perl

File: repack [hunk 1]
```diff
@@ -15,11 +15,12 @@ use warnings;
 
 use File::Temp qw/ tempdir /;
 use File::Path qw(make_path);
+use File::Copy qw(move copy);
```
Comment (DStape) [inline repack:18]: Being very pedantic here, but could you make these alphabetical in increasing order, and normalize the use of brackets vs forward-slashes, please?

**Review by DStape — APPROVED** [review]

File: repack (hunk 1 above)
Comment (JammyStuff) [reply repack:18]: @bluca Once you've addressed this comment, I'm happy to merge.

**Comment by bluca** [comment]
I'll try to go with Jan's suggestion to hack build-iso auto/config instead for now as there's less churn, if that fails will reopen this

**Pull request closed without merging**